mizar/obj/droplet.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, name, obj_api, opr_store, spec=None):
         self.known_bouncers = {}
         self.known_nets = {}
         self.known_eps = {}
+        self.known_ftn = {}
```

Comment:
See my comment on using the store object instead. All what you need then is to have one field 
```
self.is_ftn
```